manual/src/threelang/tfgrid/vm.md

```diff
@@ -0,0 +1,53 @@
+# VM Namespace
+
+- To deploy a number of virtual machines on the same network, use the vm namespace.
+
+## Create Operation
+
+- action name: !!tfgrid.vm.create
+- parameters:
+  - network [required]
```

Comment:
This can be optional. What if a user wants just one VM, then he doesn't care about the network part. In that case choose some random network name. And all vms that are added in the same session that don't have the network parameter set should be added to the same network (aka set the network parameter yourself to a random network that you choose).